doc/contribute/guidelines.rst

```diff
@@ -554,6 +563,31 @@ exceptions:
 * Avoid using binary literals (constants starting with ``0b``).
 * Avoid using non-ASCII symbols in code, unless it significantly improves
   clarity, avoid emojis in any case.
+* Use proper capitalization of nouns in code comments (e.g. ``UART`` and not
+  ``uart`` or ``CMake`` ``cmake``).
+
+Beyond C code, the following coding style rules apply to other types of files:
+
+* CMake
+
+  * Indent with spaces, indentation is two spaces.
+  * No space between commands (e.g. ``if``) and the corresponding opening
+    bracket (e.g. ``(``).
+
+* Devicetree
+
+  * Indent with tabs.
+  * No additional empty lines after ``};``.
+  * Single empty line between elements.
```

Comment:
Elements? Nodes? Does this address my concern above? If so, I think we need clearer intention here.

Reply:
@nordicjm @gmarull @bjarki-andreasen

Reply:
cc devicetree collaborators... @decsny @galak @rruuaanng

Reply:
similarly here, reword to:
```
* Single empty line between same level `};`
```